README.md

```diff
@@ -100,6 +100,7 @@ as such the below table reference which versions are compatible.
 | 0.28.0            | 2.22.X         |
 | 0.29.0            | 2.23.X         |
 | 0.30.0            | 2.24.X         |
+| 0.31.0            | 2.25.X         |
 
 
 ## Missing Functionality
```

Comment:
What do you mean? That I need to do more than copy paste the previous release updates? :sweat_smile:
Let me remove this part and read the other sections too in case we need to update things there too.